package.json

```diff
@@ -25,7 +25,9 @@
     "lint": "eslint .",
     "lint:fix": "eslint . --fix",
     "pretest": "yarn run lint",
-    "test:debug": "react-scripts --inspect-brk test --runInBand --no-cache"
+    "test:debug": "react-scripts --inspect-brk test --runInBand --no-cache",
+    "coverage": "react-scripts test --env=jsdom --coverage",
+    "coveralls": "cat ./coverage/lcov.info | node node_modules/.bin/coveralls"
```

Comment:
I might prefer to see this as `test:report_coverage` or something of the sort. The idea being to both make clearer what this script is doing and not unnecessarily couple our interface to that of our coverage tracking tool.

There are definitely counterarguments to be made.

Reply:
makes sense. Will make updates

Reply:
I have updated the ticket to have `yarn test:coverage` to generate the coverage report.